com.github.unrud.djpdf.yaml

```diff
@@ -482,8 +482,8 @@ modules:
       - --enable-unittest=no
     sources:
       - type: archive
-        url: https://libopenraw.freedesktop.org/download/exempi-2.6.2.tar.bz2
-        sha256: 4d17d4c93df2a95da3e3172c45b7a5bf317dd31dafd1c7a340169728c7089d1d
+        url: https://libopenraw.freedesktop.org/download/exempi-2.6.3.tar.bz2
+        sha256: b0749db18a9e78cf771737954a838cdcdb1d5415888bac1ba9caf8cba77c656c
```

Comment:
the shell source below can be removed as the tarball now have recent config.guess and config.sub.

(it's harmless otherwise)